examples/node-groups/windows-node-groups/main.tf

```diff
@@ -51,7 +51,7 @@ module "eks_blueprints" {
     mng_spot_medium = {
       node_group_name = "mng-spot-med"
       capacity_type   = "SPOT"
-      instance_types  = ["t3.large", "t3.xlarge"]
+      instance_types  = ["t3.large", "m6i.large", "m6a.large", "m5n.large"]
```

Comment:
what is the reason for the instance type changes?

Reply:
Refer to [Cluster Autoscaler best practices](https://aws.github.io/aws-eks-best-practices/cluster-autoscaling/#configuring-your-node-groups). If instance types with different CPU and memory capacity are used for horizontal autoscaling, resources may get wasted.